weave/env.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
We already have environment.py. Should we just use that?

Reply:
modified in next commit